packages/docs/composables/useRecaptcha.ts

```diff
@@ -0,0 +1,41 @@
+import {ref} from 'vue';
+import { isServer } from 'vuestic-ui/src/utils/ssr'
+
+let addedScript = false;
+
+interface Recaptcha {
+  render(
+    container: string | HTMLElement,
+    parameters: {
+      sitekey: string;
+      theme?:	'dark' | 'light';
+      size?:	'compact' | 'normal';
+      tabindex?: number;
+      callback?: (token: string) => void;
+      'expired-callback'?: () => void;
+      'error-callback'?: () => void;
+    }
+  ): string;
+  reset(widgetId?: string): void;
+}
+
+const recaptcha = ref<Recaptcha>();
```

Comment:
Could you run linter with --fix? We generally don't use `;` in vuestic.

Reply:
Linter doesn't enforce consistent colon code style unfortunately